Lib/importlib/resources/_adapters.py

```diff
@@ -35,7 +35,7 @@ def _io_wrapper(file, mode='r', *args, **kwargs):
     elif mode == 'rb':
         return file
     raise ValueError(
-        f"Invalid mode value '{mode}', only 'r' and 'rb' are supported"
```

Comment:
This looks like something got reverted. Gotta track it down and backport it.

Reply:
Looks like #98005

Reply:
I've cherry-picked that PR to importlib_resources and merged it with cpython. I'll just revert this change in this file.

Reply:
Done in 1637ad48b5